tests/MongoDB.Driver.Tests/Specifications/server-discovery-and-monitoring/tests/integration/connectTimeoutMS.json

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
Refresh this file from the spec repo as it has changed since being pulled in. Notably `hello` was added to the list of fail commands.